README.md

```diff
@@ -14,7 +14,41 @@ Accessible modal dialog component for React.JS
   <p>Etc.</p>
 </Modal>
 ```
+## Styles
+Styles are passed as an object with 2 keys, 'overlay' and 'content' like so
+```js
+{
+  overlay : {
+    position          : 'fixed',
+    top               : 0,
+    left              : 0,
+    right             : 0,
+    bottom            : 0,
+    'backgroundColor' : 'rgba(255, 255, 255, 0.75)'
+  },
+  content : {
+    position                   : 'absolute',
+    top                        : '40px',
+    left                       : '40px',
+    right                      : '40px',
+    bottom                     : '40px',
+    border                     : '1px solid #ccc',
+    background                 : '#fff',
+    overflow                   : 'auto',
+    'WebkitOoverflowScrolling' : 'touch',
```

Comment:
FYI, actually reviewing your PR on the react-modal repo but figured I could leave comments here instead... Just got down to the actual code section and noticed keys aren't quoted there so you can remove the quotes from the sample code.